README.md

```diff
@@ -1,34 +1,70 @@
 # DSS deployment scripts
 
-A set of scripts that use [DSS deploy](https://github.com/makerdao/dss-deploy) to deploy the DSS system in a testchain, testnets and mainnet.
+A set of scripts that deploy [dss](http://github.com/makerdao/dss) to an
+Ethereum chain of your choosing.
 
-### TODO
+## Description
 
-- More cases to test scenarios for testchain script
-- Mainnet and other testnets scripts
+This repo is composed of two steps:
 
-## Additional Documentation
+* [smart contracts](./contracts) (imported as submodules) that deploy a base dss
+* Bash [scripts](/scripts) to modify the state of the base system
 
-- `dss-deploy` [source code](https://github.com/makerdao/dss-deploy)
-- `dss` is documented in the [wiki](https://github.com/makerdao/dss/wiki) and in [DEVELOPING.md](https://github.com/makerdao/dss/blob/master/DEVELOPING.md)
+At the end of the first step, the addresses of deployed contracts are written to
+an `out/addresses.json` file. The scripts read those addresses and use `seth`
+and `dapp` to modify the deployment, using the values in `out/config.json`.
 
-## Deployment
+## Installing
+
+The preferred way to install everything necessary to deploy is Nix. Run
+
+```
+$ nix-shell --pure
+```
 
-### Prerequisites:
+to drop into a Bash shell with all dependency installed.
 
-If you use nix, run `nix-shell` to drop into a shell with all dependencies
-installed.
+### Non-nix
 
-Otherwise:
+If you don't use Nix, you'll need to install:
```

Comment:
Why do we still support this?  Peeps will have to install `nix` to install dapptools anyway?

Reply:
I'm fine with removing mentions of non-Nix, as I think since a couple of releases we're releasing to Kovan with Nix.

@zdumitru can you confirm?
@wkampmann are there any lingering doubts about Nix being humanity's final answer to the dependency problem?

Reply:
I personally use it without nix, for me it is just faster to test.
Dumitru used nix for deploying 0.2.7, but due some issue last week, he deployed 0.2.8 using the direct method (pending to see with @icetan)